src/operators_dense.jl

```diff
@@ -20,6 +20,9 @@ end
 Operator{BL,BR}(basis_l::BL,basis_r::BR,data::T) where {BL,BR,T} = Operator{BL,BR,T}(basis_l,basis_r,data)
 Operator(basis_l::BL,basis_r::BR,data::T) where {BL,BR,T} = Operator{BL,BR,T}(basis_l,basis_r,data)
 Operator(b::Basis,data) = Operator(b,b,data)
+Operator(qet::Ket{B}...) where B = Operator(qet[1].basis, GenericBasis(length(qet)), hcat(getfield.(qet,:data)...))
+Operator(basis_r::Basis,qet::Ket{B}...) where B = Operator(qet[1].basis, basis_r, hcat(getfield.(qet,:data)...))
```

Comment:
I believe this is raising an "unbound type parameter" error in the tests. Check the failing reports. The easiest way to fix that is probably simply dropping the `where B`, i.e. just use `qet::Ket...`. For more about this you can read https://github.com/JuliaTesting/Aqua.jl/issues/87

Reply:
I realize now that a method for Operator with no input at all is a bit weird.
Changed it so it has to get a least one input state.